config/04-kubernetes/40-auth-ingress.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     cert-manager.io/cluster-issuer: letsencrypt-prod
     dns.gardener.cloud/dnsnames: 'auth.hub.tekton.dev'
     dns.gardener.cloud/ttl: "3600"
-  name: auth
+  name: tekton-hub-auth
```

Comment:
can you please modify the line https://github.com/tektoncd/hub/blob/0937bdd758edcafc8599859c60074a6e9660cfa2/config/04-kubernetes/40-auth-ingress.yaml#L16 as well?